rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -111,6 +111,10 @@ def to_exported_dict(
             exported_dict['cost_basis_taxable'] = taxable_basis
             exported_dict['cost_basis_free'] = free_basis
             exported_dict['asset'] = str(self.asset)
+            try:
+                exported_dict['asset_symbol'] = self.asset.symbol_or_name()
+            except UnknownAsset:
+                exported_dict['asset_symbol'] = ''
```

Comment:
Can you change the keys for both things here and elsewhere in code and tests?

The reason is to specifcy the thing above is the id. And the other key is the human readable text. Could also do `asset_symbol_or_name` but perhaps that's too long

```suggestion
            exported_dict['asset_identifier'] = str(self.asset)
            try:
                exported_dict['asset'] = self.asset.symbol_or_name()
            except UnknownAsset:
                exported_dict['asset'] = ''
```